src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -231,26 +232,26 @@ public void clickSectionCheckBoxInSectionLevel(int instrNum, int sectionLevelNum
     }
     
     public void clickViewStudentCheckBoxInSectionLevel(int instrNum, int sectionLevelNum) {
-        String cssSelector = "#tuneSectionPermissionsDiv" + sectionLevelNum + "ForInstructor" + instrNum
-                + " > div > div.panel-body > div.col-sm-6.border-right-gray > input[type=\"checkbox\"]:nth-child(1)";
+        String cssSelector = "input[type=\"checkbox\"][name=\"canviewstudentinsectionsectiongroup"
+                             + sectionLevelNum + "\"]";
```

Comment:
Can I just confirm that this works even without the `instrNum` because when editing permissions for one instructor, the checkboxes for all other instructors will be hidden, and thus cannot be clicked?

In any case, I think it is better to include the `instrNum` in the CSS selector so there will be an test failure if a test tries to click the checkbox for the wrong instructor